src/lib/cache/subCache.cpp

```diff
@@ -77,6 +77,8 @@ EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityT
 
   if (isPattern)
   {
+    // FIXME P5: recomp error should be captured? have a look to other usages of regcomp()
+    // in order to see how it works
```

Comment:
I would create it but with a broader scope. I'd like to include an anaylsis of std::regex in order to see if using it will make regex management simpler (we have had many problems with them in this PR, suposed to be simple from a functionaly point of view).